### PR TITLE
GUI: Add a checkbox to allow overriding the global backend settings

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: gui/widget.h [hunk 1]
```diff
@@ -476,6 +476,15 @@ class OptionsContainerWidget : public Widget {
 	 */
 	virtual bool save() = 0;
 
+	/** Implementing classes should return if there are relevant keys set in the configuration domain
+	 *
+	 * @return true if there relevant are keys set in the configuration domain
```
Comment (antoniou79) [inline gui/widget.h:481]: This is a minor thing, but the comment needs a syntax correction to:
"true, if there are relevant keys set in the configuration domain"
as in the line above.

Comment (ccawley2011) [reply gui/widget.h]: Fixed, thanks.

**Comment by sev-** [comment]
The change is fine, however, it got bitrot. @ccawley2011 could you please rebase it?

**Comment by ccawley2011** [comment]
> The change is fine, however, it got bitrot. @ccawley2011 could you please rebase it?

OK, done.

**Comment by Mataniko** [comment]
Please also update/add the relevant docs for this GUI option

**Comment by ccawley2011** [comment]
> Please also update/add the relevant docs for this GUI option

I added a brief description of the Backend tab as part of PR #2832.

**Comment by bluegr** [comment]
Thanks for rebasing this! I too like this change, and it's in good shape to me merged.

Merging

**Pull request merged**